test/google-youtube-custom-thumbnail.html

```diff
@@ -4,12 +4,12 @@
   <head>
     <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
     <title>google-youtube Basic Tests</title>
-    <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
-    <script src="../../web-component-tester/browser.js"></script>
+    <script src="../../../@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
+    <script src="../../../wct-browser-legacy/browser.js"></script>
 
-    <link rel="import" href="../../polymer/polymer.html">
-    <link rel="import" href="../../test-fixture/test-fixture.html">
-    <link rel="import" href="../google-youtube.html">
+    <script type="module" src="../../../@polymer/polymer/polymer-legacy.js"></script>
+    <script type="module" src="../../../@polymer/test-fixture/test-fixture.js"></script>
+    <script type="module" src="../google-youtube.js"></script>
```

Comment:
delete this block of script type modules

Reply:
Done